# goal
string handle_type
geometry_msgs/PoseStamped handle_pose
---
# result
bool success
---
# feedback
string current_state
string message
